opal/corelib/enumerable.rb

```diff
@@ -177,6 +177,10 @@ def count(object = undefined, &block)
   end
 
   def cycle(n = nil, &block)
+    if `arguments.length > 1`
```

Comment:
in cRuby it's < 0 as well:

    ruby -e "p method(def a(b=1) end).arity" # => -1